lib/CL/clUnloadPlatformCompiler.c

```diff
@@ -22,11 +22,25 @@
  **/
 
 #include "pocl_cl.h"
+#include "pocl_llvm.h"
 
 CL_API_ENTRY cl_int CL_API_CALL
 POname(clUnloadPlatformCompiler)(cl_platform_id platform)
 CL_API_SUFFIX__VERSION_1_2
 {
+#if defined(BUILD_ICD) && defined(OCS_AVAILABLE)
```

Comment:
If I understood correctly, an OpenCL implementation can provide multiple platforms even without ICD and if not, it should return the single platform ID if BUILD_ICD is not on.